storage/innobase/include/log0files_io.h

```diff
@@ -562,6 +562,22 @@ inline uint32_t log_block_convert_lsn_to_hdr_no(lsn_t lsn) {
          static_cast<uint32_t>(lsn / OS_FILE_LOG_BLOCK_SIZE % LOG_BLOCK_MAX_NO);
 }
 
+#ifdef XTRABACKUP
+/** convert hdr to LSN
+@param[in] header		log_block_number
```

Comment:
better to use header_num or hdr_num or hdr_no

Reply:
fixed